app/assets/stylesheets/application.scss

```diff
@@ -14,5 +14,6 @@
 // Layouts
 
 // Components
+@import 'components/footer';
```

Comment:
We don't make components for the footer. We use a layout css file to design the whole layout style. The footer style should be inside the layout stylesheet. Check this [layout file](https://github.com/nimblehq/mia-web/blob/development/engines/mia_account/app/assets/stylesheets/mia_account/layouts/_default.scss#L73)  for reference.

Also check out CSS structure guideline 👍  https://nimblehq.co/compass/development/code-conventions/css/#stylesheets-structure